docs/man/docker-images.1.md

```diff
@@ -34,7 +34,9 @@ versions.
    Show all images (by default filter out the intermediate image layers). The default is *false*.
 
 **-f**, **--filter**=[]
-   Provide filter values (i.e., 'dangling=true')
+   Provide filter values. Valid filters:
```

Comment:
Ok, gotcha.  

-->

Filters the output. The `dangling=true` filter finds unused images. While `LABEL=foo.com=amd64` filters for images with a `foo.com` value of `amd64`. The `LABEL==` filter finds images without labels.